### PR TITLE
Rebase IPFS to v0.4.19

### PR DESCRIPTION
**Pull request by cpacia**

### CODE REVIEW

File: ipfs/identity_test.go [hunk 1]
```diff
@@ -9,7 +9,7 @@ import (
 	"github.com/tyler-smith/go-bip39"
 )
 
-var keyHex = "08011260499228645d120d15b5008b1da0b9dba898df328001ea03c0be84a64c41d205ff1b8339a303cd8cf2945b66c89ac29fa90e79731d67000694284791af404eeb1f1b8339a303cd8cf2945b66c89ac29fa90e79731d67000694284791af404eeb1f"
```
[the remark this answers is not in the document]
Comment (cpacia) [reply ipfs/identity_test.go]: They changed the protobuf serialization of the key. It broke our test. Shouldn't break the functionality though.

File: schema/manager.go [hunk 2]
```diff
@@ -27,6 +29,25 @@ const (
...
+			panic(err)
+		}
+
+		err = loader.Inject()
+		if err != nil {
+			panic(err)
+		}
+	})
+}
+
```
[the remark this answers is not in the document]
Comment (cpacia) [reply schema/manager.go]: Yeah the plugin is for the database. I'm using the code the same way they use it. I agree it's weird it's just using "" but that's just how they do it.

File: ipfs/identity_test.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (placer14) [reply ipfs/identity_test.go]: We don't use that protobuf serialization anywhere for longterm storage, right?

[the remark this answers is not in the document]
Comment (placer14) [reply ipfs/identity_test.go]: Assuming we don't use it that serialization anywhere, this appears to be GTM

**Pull request merged**